helpers/z_schema.js

```diff
@@ -49,27 +49,15 @@ z_schema.registerFormat('username', function (str) {
 });
 
 z_schema.registerFormat('hex', function (str) {
-	try {
-		Buffer.from(str, 'hex');
-	} catch (e) {
-		return false;
-	}
-
-	return true;
+	return /^[a-fA-F0-9]*$/.test(str);
```

Comment:
also, we are not checking length anywhere, what do you think about checking length if it fits in 32 or 64? I think we are not testing any different scenarios.